tiledb/sm/serialization/query.cc

```diff
@@ -1680,17 +1680,26 @@ Status query_from_capnp(
             }
           }
 
+          // attr_copy_state==nulptr models the case of deserialization code
+          // being called via the C API tiledb_query_deserialize (e.g. unit
+          // test serialization wrappers).
+          // When a rest_client exists, there is always a copy_state
```

Comment:
copy_state -> attr_copy_state

Reply:
If you fix here, fix below.

Reply:
Clarified the comment in both places.

Reply:
it was actually intentional as the caller code is actually passing copy_state originally.